lib/init/passport.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+/**
+ * @fileoverview Passport authentication init
+ */
+const config = require('config');
+const passport = require('passport');
+const LocalStrategy = require('passport-local').Strategy;
+const bcrypt = require('bcrypt-nodejs');
+
+const acl = require('../auth/acl');
+
+const database = require('../database');
+
+// Passport Authentication setup
+passport.use(new LocalStrategy(
+  {
```

Comment:
God this is odd. We don't follow this pattern anywhere else. I don't know what to do, though. Maybe `LocalStrategy({ usernameField: 'email' }, (username, password, cb) =>`